docs/pact-properties.md

```diff
@@ -57,7 +57,7 @@ transaction has the proper signatures to satisfy the keyset named `admins`:
 ```lisp
 (defun read-account (id)
   @doc   "Read data for account ID"
-  @model (properties [(authorized-by 'admins)])
+  @model [(property (authorized-by 'admins))])
```

Comment:
```suggestion
  @model [(property (authorized-by 'admins))]
```

Reply:
Here and elsewhere, the rest seem to have the trailing `)` as well.

Reply:
Fixed -- thank you!